gitlab/resource_gitlab_project_level_mr_approvals_test.go

```diff
@@ -0,0 +1,152 @@
+package gitlab
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	gitlab "github.com/xanzy/go-gitlab"
+)
+
+func TestAccGitlabProjectLevelMRApprovals_basic(t *testing.T) {
+
+	var projectApprovals gitlab.ProjectApprovals
+	rInt := acctest.RandInt()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckGitlabProjectLevelMRApprovalsDestroy,
+		Steps: []resource.TestStep{
+			{
+				SkipFunc: isRunningInCE,
+				Config:   testAccGitlabProjectLevelMRApprovalsUpdateConfig(rInt),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckGitlabProjectLevelMRApprovalsExists("gitlab_project_level_mr_approvals.foo", &projectApprovals),
+					testAccCheckGitlabProjectLevelMRApprovalsAttributes(&projectApprovals, &testAccGitlabProjectLevelMRApprovalsExpectedAttributes{
+						resetApprovalsOnPush:                      true,
+						disableOverridingApproversPerMergeRequest: true,
+						mergeRequestsAuthorApproval:               true,
+						mergeRequestsDisableCommittersApproval:    true,
+					}),
+				),
+			},
+		},
+	})
+}
+
+func TestAccGitlabProjectLevelMRApprovals_import(t *testing.T) {
+	resourceName := "gitlab_project_level_mr_approvals.foo"
+	rInt := acctest.RandInt()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckGitlabProjectLevelMRApprovalsDestroy,
+		Steps: []resource.TestStep{
+			{
+				SkipFunc: isRunningInCE,
+				Config:   testAccGitlabProjectLevelMRApprovalsUpdateConfig(rInt),
+			},
+			{
+				SkipFunc:                isRunningInCE,
+				ResourceName:            resourceName,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"project_id"},
```

Comment:
You should be able to remove this line once you fix the "projectId" typo in your read funcyion (see my other comment).